sim/create.ts

```diff
@@ -33,7 +37,7 @@ export type Transport = {
   address(): string;
   name(): string;
   setInfo(info: Info): void;
-  //  getInfo(): Promise<Info>;
+  // getInfo(): Promise<Info>;
```

Comment:
Another whitespace only change

Reply:
That was not prettier but me, because only one space is normal, right?